weedcoco/cwfid_to_json/cwfid_imageinfo.json

```diff
@@ -17269,6 +17269,7 @@
     ],
     "categories": [
         {
+            "name": "crop: daugus carota",
```

Comment:
This might be over thinking it, but I feel the "name" field name might be a bit misleading. Really what we are capturing is "identity" as a combo of role+name.

Reply:
It matches the semantics of "name" in COCO: a meaningful and unique string-based identifier for each category.

Reply:
Of course, that makes sense. A hypothetical front end for users entering their data could always just assemble the name field from a tick box for role and a free text entry for the species

Reply:
> a free text entry for the species

or an EPPO-guided entry!